src/relay/backend/contrib/cmsisnn/relay_to_tir.cc

```diff
@@ -757,7 +778,10 @@ class RelayToTIRVisitor : public MixedModeMutator {
   static constexpr int32_t kScaledDiffIntegerBits = 5;
   static constexpr int32_t kInputBits = 5;
   static constexpr double kBeta = 1.0;
+  /*! \brief Unique id for context buffer needed by CMSIS-NN layers. */
   int32_t context_buffer_id_;
+  /*! \brief Skip arguments in the call to global partitioned function. */
+  std::list<int32_t> skip_call_args_;
```

Comment:
list find complexity is O(n), unordered_set should be a better replacement.

Reply:
It was supposed to have only 2 elements. But, I have modified it now.